### ISSUE TITLE
Carving station turns all vanilla stones into stone

### ISSUE THREAD

**Issue by hoolatehip** [GlobbyPotato/Rockhounding_Rocks#18]
Carving station turns granite, andesite, etc. into stone. 

Expectation: Granite + Bricks scheme = Granite Bricks
Reality: Granite + Bricks scheme = Stone bricks

**Comment by GlobbyPotato** [comment]
Hi, I cannot reproduce this issue, all the vanilla blocks turn into their respective bricks here. Is it in a pack and are latest versions?

**Comment by hoolatehip** [comment]
Yes, this is in [Eat Everything](https://www.curseforge.com/minecraft/modpacks/eat-everything) using the latest version of Rockhounding for 1.12.2. Nothing in the pack looks like it should be causing a problem here.
